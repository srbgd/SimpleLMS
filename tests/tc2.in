

drop
register librarian --login l --password pass --name test --address innopolis --phone-number 88005553535 --card-number 1234
register student --login p --password pass --name test --address innopolis --phone-number 88005553535 --card-number 9876
login
p
pass
checkout 2
exit
