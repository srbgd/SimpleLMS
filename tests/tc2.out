Success
True
True
True
Success
False
